README.md

```diff
@@ -98,7 +103,18 @@ Each time, you will (re)create container, connected to container devices will as
 For example:
 ```
 $ docker run --privileged -d -p 4723:4723 -v ~/.android:/root/.android -v /dev/bus/usb:/dev/bus/usb --name container-appium appium/appium
-``` 
+```
+
+## Connect to Multiple Android Devices on Linux
+
+The parameter `-v /dev/bus/usb:/dev/bus/usb` doesn't work for multi-device because it siphons off every usb device to a single container. Every device will to be connected to the same Appium server which doesn't work well in practice.
```

Comment:
Any reason why do you suggest that ` same appium server which doesnt work well in practice`? We have been running multiple tests on single appium server hooked to multiple devices without any issues for along time.

Reply:
Hi, I was referring to plugging in new devices with the Hub and the old Appium issues, but I do agree that the paragraph should be removed. I'll remove it and fix the conflict soon.

Reply:
Sure fix this

Reply:
Hey I fixed it yesterday, please look at `Files changed`.